source/math/FloatMatrix.ooc

```diff
@@ -45,6 +45,36 @@ FloatMatrix : cover {
 		result
 	}
 
+	setColumn: func (x: Int, values: ...) {
```

Comment:
@sebastianbaginski What if the width is 1, wouldn't `memcpy` be pretty slow in that case?